containers/kraken.yml

```diff
@@ -24,7 +24,7 @@ spec:
               name: config
             - mountPath: "/root/kraken/config"
               name: kraken-config
-            - mountPath: "/root/kraken/scenarios"
+            - mountPath: "/root/kraken/scenarios/openshift"
```

Comment:
Using the scenario locations that come as defaults in the config/config.yaml file I am getting the below error. I have a[ PR open](https://github.com/redhat-chaos/krkn/pull/337) for this to be able to add all scenario files no matter where the user might put theirs if they are doing a custom scenario




```
Traceback (most recent call last):
File "/root/kraken/run_kraken.py", line 429, in <module>
main(options.cfg)
File "/root/kraken/run_kraken.py", line 219, in main
pod_scenarios.container_run(
File "/root/kraken/kraken/pod_scenarios/setup.py", line 76, in container_run
with open(container_scenario_config[0], "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'scenarios/openshift/container_etcd.yml'
```

Reply:
Thoughts on closing this PR in favor of https://github.com/redhat-chaos/krkn/pull/337?